content/blog/monkey-patch.md

```diff
@@ -0,0 +1,62 @@
++++
+title = "Monkey Patching"
+slug = "monkey-patch"
+date = 2023-10-17T18:29:54+05:30
+image = "/images/2023/10/monkeypatch.jpg"
+draft = false
+authors = ["Nitin Rajkumar"]
+description = "Monkey patching and it's sideeffects in Ruby"
+tags = ["Software Craftsmanship", "GraphQL"]
+categories = ["Software Craftsmanship", "Ruby", "Monkey Patching"]
+type = ""
++++
+
+Ruby, being a dynamic programming language provides the ability and hence freedom to reopen the existing classes and change their behavior during runtime by adding new methods or modifying existing methods. 
+
+#### How to monkey patch
+
+```ruby
+class Sound
+	def car
+		"honk"
+	end
+end
+class Sound
+	def bike
+		"peep-peep"
+	end
+end
```

Comment:
use space instead of tab for indentation and use 2 space indentation. it will look better and will also align with indentation used at line 43 to 47
```suggestion
class Sound
  def car
    "honk"
  end
end
class Sound
  def bike
    "peep-peep"
  end
end
```